### PR TITLE
App layout refactoring and JWT authentication

### PR DESCRIPTION
**Pull request by akx**
h/t https://github.com/City-of-Helsinki/respa/commit/a01bf392ea2ab41ef9b50f8de8e536e625c617da

### CODE REVIEW

**Comment by akx** [comment]
Closing -- will need to rejiggle.

File: kerrokantasi/urls.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.conf import settings
+from django.conf.urls import include, url
+from django.conf.urls.static import static
+from django.contrib import admin
+from kk import urls_v1
+
+urlpatterns = [
+    url(r'^v1/', include(urls_v1)),
+    url(r'^admin/', include(admin.site.urls)),
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (rphlo) [inline kerrokantasi/urls.py]: Django should only serve static when in dev or debug is True

Comment (akx) [inline kerrokantasi/urls.py]: Fixed.

File: kerrokantasi/settings/util.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+def load_secret_key(settings):
+    """
+    Load a secret key from `.django_secret` if one is not already set.
+
+    :param settings: Settings dict
+    :type settings: dict
+    """
+    if settings.get("SECRET_KEY"):
+        return
+    secret_file = os.path.join(settings.get("BASE_DIR"), '.django_secret')
```
Comment (rphlo) [inline kerrokantasi/settings/util.py]: Adding .django_secret to .gitignore just in case...

Comment (akx) [inline kerrokantasi/settings/util.py]: Point. Added in the PR too.

**Pull request merged**